frontend/src/components/graph/GraphView3D.tsx

```
@@ -1,73 +1,597 @@
+    updateDimensions();
+
+    const resizeObserver = new ResizeObserver(updateDimensions);
+    resizeObserver.observe(container);
+
+    return () => resizeObserver.disconnect();
+  }, []);
+
   const nodeMap = useMemo(() => {
     const map = new Map<string, Graph3DNode>();
-    data.nodes.forEach(node => map.set(node.node_id, node));
+    data.nodes.forEach((node) => map.set(node.node_id, node));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)